.circleci/config.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           install-npm: false
           node-version: << parameters.node-version >>
       - run: npm install
+      - run: npm run prepare
```

Comment:
@scottbell this _should_ not be necessary and will need to be performed on several other CI stages if it is.

Reply:
The `dist` directory seems to be missing without it. If we can get that another way, so shared workers can load in tests, that would be great!

Reply:
environment:
      NODE_ENV: development